### ISSUE TITLE
Bounding rectangle doesn't fit all device types

### ISSUE THREAD

**Issue by kylenahas** [missionpinball/mpf-monitor#31]
As pointed out by Avery on the Google Groups [feedback thread](https://groups.google.com/g/mpf-users/c/Ybg09FXLDaU/m/qbId9UPrBgAJ), devices are clipped improperly when the playfield window is not active.

![image](https://user-images.githubusercontent.com/716422/90449455-57284b80-e09c-11ea-9786-1d4c2e7354bd.png)

I believe the following code to be at fault: 
```
    def boundingRect(self):
        return QRectF(self.device_size / -2, self.device_size / -2,
                      self.device_size, self.device_size)
```
[➔ playfield.py](https://github.com/missionpinball/mpf-monitor/blob/827e329008d3c7bcf4bb65580e7904b9c7bb3fb1/mpfmonitor/core/playfield.py#L138:L140)

**Comment by atummons** [comment]
I agree.  I had something kind of working, but it wasn't what I wanted.  Setting a bouncing rect is exactly that...a rectangle.  So a triangle shape has a clickable area that of a rectangle.  That is a bit of an issue there, but that might just be a limitation.

Further when you start changing angles, the rectangle angle doesn't adjust.  So you have to import cos and sin to find a bounding rect.  We would really want something like this: https://www.mathopenref.com/coordbounds.html where explicit coordinates are given relative to the center point.  I will do some more digging on this one.

**Comment by bosh** [comment]
I think I solved it using the simple min/max x/y coord bounds algorithm as seen in the previous post link.
https://github.com/missionpinball/mpf-monitor/pull/47
